collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -430,6 +430,7 @@ private void setupViewModels() {
                 .of(this, changesReasonViewModelFactory)
                 .get(FormSaveViewModel.class);
 
+        formSaveViewModel.setAnalytics(analytics);
```

Comment:
This could be constructor dependency right? Then there doesn't need to be any nullability risk in the object.

Reply:
Right, done.